linalg/ode.hpp

```diff
@@ -91,6 +91,22 @@ class ODESolver
       while (t < tf) { Step(x, t, dt); }
    }
 
+   /// Function for getting and setting the state vectors
+   virtual int   GetMaxStateSize() { return 0; }
+   virtual int   GetStateSize() { return 0; }
+   virtual void  GetStateVector(int i, Vector &state)
+   {
+      mfem_error("ODESolver has no state vectors");
+   }
+   virtual const Vector &GetStateVector(int i)
```

Comment:
Appveyor is upset that this function doesn't return anything.

Reply:
We probably can just return an empty temporary here: something like `return Vector();` or `Vector s; return s;`

Reply:
In both cases you get a warning  

> -Wreturn-local-addr

I went with `Vector *s = NULL; return *s;`